cypress/integration/t-3.js

```diff
@@ -0,0 +1,26 @@
+import LoginPage from '../objects/loginPage';
+import TasksPage from '../objects/tasksPage';
+import CreateTaskPage from '../objects/createTaskPage';
+
+const loginPage = new LoginPage();
+const tasksPage = new TasksPage();
+const createTaskPage = new CreateTaskPage();
+
+describe('Create task', function() {
+
+    beforeEach(function(){
+        cy.loginAs("Admin")
+        cy.visit('/')
```

Comment:
you could go to /tasks directly, in that case it would be less loading times, because now you go to home page and only then to /tasks.